### PR TITLE
Router

### PR DESCRIPTION
**Pull request by joanajimfer**
Hi @beatrizAdalab , 
In this branch, I have created a Router with Links to Main and Detail Pages.
I create both components to (Main and Detail), so we could cluster all the little components in these two pages that come from App. 
I pull here the detail_hours branch.
Please, check it because for me in the mobile layout for Detail Page the Daily component appears repeated and I do not understand why. In the horizontal view, it does not happen.

### CODE REVIEW

File: src/components/DailyDetail/index.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+               <HourData hour={20} />
+               <HourData hour={22} />
+               <HourData hour={24} />
+               <HourData hour={2} />
+               <HourData hour={4} />
+               <HourData hour={6} />
+            </table>
+        )
+    }
+}
```
Comment (beatrizAdalab) [inline src/components/DailyDetail/index.js:34]: We have to be careful. the API will return the information every three hours.

**Review by beatrizAdalab — APPROVED** [review]
Good job! you have to check the data that the API returns, the daily information is every three hours.

File: src/components/DailyDetail/index.js (hunk 1 above)
Comment (joanajimfer) [reply src/components/DailyDetail/index.js:34]: Ok, I will correct it.

**Comment by joanajimfer** [comment]
I close this PR because now we are not going to use Routes in the web

**Pull request closed without merging**